src/components/AlertDisplay/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
+import alertLevels from 'ringcentral-integration/modules/Alert/alertLevels';
+
 import dynamicsFont from '../../assets/DynamicsFont/DynamicsFont.scss';
 import styles from './styles.scss';
 
-// TODO animation
-
-function Message(props) {
+export function Message(props) {
```

Comment:
Due to the tooling we use for automatically generating some examples, please only export 1 component per file. If a component is worthy of exporting, please separate it into its own file.